src/main/java/frc/robot/RobotContainer.java

```diff
@@ -31,32 +33,35 @@
 import frc.robot.subsystems.Pivot;
 import frc.robot.subsystems.Shooter;
 import frc.robot.subsystems.Swerve;
+import frc.robot.subsystems.LED;
 
 /*
  * Instantiates all robot subsystems and button commands.
 */
 public class RobotContainer {
-
-    Swerve swerveSubsystem;
+    public Swerve swerveSubsystem;
     Shooter shooterSubsystem;
     public Pivot pivotSubsystem;
     public Climber climberSubsystem;
     public Intake intakeSubsystem;
     Controllers controllers;
     public AutoDriveBuilder autoDriveBuilder;
     public AutoShootBuilder autoShootBuilder;
-
+    public LED ledSubsystem; 
```

Comment:
why is this public? The rest that are public are either used as a quick bodge in Robot.java or in AutoSelector.java